### ISSUE TITLE
$minErr module loading error can cause truncated URLs

### ISSUE THREAD

**Issue by jeffbcross** [angular/angular.js#10285]
Since the 2nd minErr argument is often the error's stack trace when trying to load modules, the url can often be cut off by the browser.
https://github.com/angular/angular.js/blob/bf6a79c3484f474c300b5442ae73483030ef5782/src/auto/injector.js#L756

This message should be replaced with something more intelligent, ideally showing which dependent module was missing or broken.

**Comment by petebacondarwin** [comment]
If the call stack is very long then the URL can also be very long. This is common when there are error messages in loading modules and in infinite digests. Also I think there is often some kind of recursion in the messages where you have to click through a number of times to get to the actual error message.

TLDR; improve the error messages from the core.

**Comment by Narretz** [comment]
Depending on what causes it, the error will already show context. Otherwise this is just a special case of https://github.com/angular/angular.js/issues/14744
